u19_pipeline/imaging_pipeline.py

```diff
@@ -15,55 +24,41 @@
 3. Utility functions
     + get_imaging_root_data_dir()
     + get_scan_image_files()
-    + get_suite2p_dir()
+    + get_processed_dir()
 
 For more detail, check the docstring of the element:
     help(scan_element.activate)
     help(imaging_element.activate)
-
 """
 
-# 1. Schema names --------------------------------------------------------------
-import datajoint as dj
-import pathlib
-
-from u19_pipeline import acquisition, subject, imaging_rec
+# 1. Schema names ----------------------------------------------------------------------
+imaging_schema_name = dj.config['custom']['database.prefix'] + 'imaging_element'
+scan_schema_name = dj.config['custom']['database.prefix'] + 'scan_element'
 
-from u19_pipeline.imaging_rec import Session
-from element_calcium_imaging import scan as scan_element
-from element_calcium_imaging import imaging as imaging_element
-from element_interface.utils import find_full_path
-
-imaging_schema_name = dj.config['custom']['database.prefix'] + 'imaging_rec_element'
-scan_schema_name = dj.config['custom']['database.prefix'] + 'scan_rec_element'
-
-
-# 2. Upstream tables -----------------------------------------------------------
-# from u19_pipeline.acquisition import Session
+# 2. Upstream tables -------------------------------------------------------------------
 from u19_pipeline.reference import BrainArea as Location
 
-schema = dj.schema('u19_' + 'lab')
+Session = recording.FieldOfView
 
+lab_schema = dj.schema(dj.config['custom']['database.prefix'] + 'lab')
 
-@schema
+@lab_schema
 class Equipment(dj.Manual):
     definition = """
     scanner: varchar(32)
     """
 
-
-# 3. Utility functions ---------------------------------------------------------
+# 3. Utility functions -----------------------------------------------------------------
 
 def get_imaging_root_data_dir():
     data_dir = dj.config.get('custom', {}).get('imaging_root_data_dir', None)
     return data_dir if data_dir else None
 
-
 def get_scan_image_files(rec_process_key):
```

Comment:
This function to be modified to get tiff files from recording.Recording recording_directory

Reply:
Good catch!  Will update later this week.  Created https://github.com/datajoint-company/U19-pipeline_python/issues/34 to keep track of this issue.